### ISSUE TITLE
Cannot cancel when starting LiveTV

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#10]
```
What steps will reproduce the problem?
1. Start a LiveTV session on any channel
2. While buffering, press Cancel
3. Instead of cancelling, it finished buffering and then starts playing TV.

What is the expected output? What do you see instead?

I expect it to cancel the buffering, and not play.

What version of the product are you using? On what operating system?

658 on Linux

Please provide any additional information below.

If any more information is required, I'll add it.

```

Original issue reported on code.google.com by `daniel.a...@gmail.com` on 12 Jan 2009 at 5:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `semir.pa...@gmail.com` on 13 Jan 2009 at 5:05
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn-1510 for 0.21 backends.  0.22 backends won't get the fix until the 
next
official release of XBMC
```

Original comment by `semir.pa...@gmail.com` on 20 Oct 2009 at 2:53
- Changed state: **Fixed**
